vast.py

```diff
@@ -2179,6 +2294,11 @@ def reboot__instance(args):
         rj = r.json();
         if (rj["success"]):
             print("Rebooting instance {args.ID}.".format(**(locals())));
+            if (args.schedule):
+                cli_command = "reboot instance"
+                api_endpoint = "/api/v0" + "/instances/reboot/{id}/".format(id=args.ID)
```

Comment:
and again lol- perhaps just `api_endpoint = url` if you wanna keep your descriptive variable name

Reply:
don't want hostname:port, etc. included